doc/configuration.rst

```diff
@@ -622,6 +623,67 @@ parameter::
 If the value of ``first_notebook_cell`` or ``last_notebook_cell`` is set to
 ``None``, then no extra first or last cell will be added to the notebook.
 
+.. _notebook_images:
+
+Adding images to notebooks
+==========================
+
+When notebooks are produced, by default (``notebook_images = False``) image
+paths from the `image` directive in rST documentation blocks (not images
+generated fom code) are included in markdown in their original form. This
```

Comment:
"in their original form" could mean "using RST syntax" (very bad) or "using their original paths". I think you mean the latter, but can you update the description to clarify?